### PR TITLE
GOA-2344_Create_annotation_stress_tests

### PR DESCRIPTION
**Pull request by rantunesboreas**

### CODE REVIEW

**Comment by eddturner** [comment]
There could/should be a test for gene-product subsets, and withFrom?

**Comment by rantunesboreas** [comment]
Have enabled the aspect test.

File: solr-cores/src/main/cores/annotation/conf/schema.xml [hunk 1]
```diff
@@ -106,8 +106,8 @@
     <field name="symbol_unsorted" type="lowercase" indexed="true" stored="false" multiValued="false"/>
     <copyField source="symbol" dest="symbol_unsorted"/>
 
-    <field name="dbSubset_unsorted" type="lowercase" indexed="true" stored="false" multiValued="false"/>
-    <copyField source="dbSubset" dest="dbSubset_unsorted"/>
+    <field name="geneProductSubset_unsorted" type="lowercase" indexed="true" stored="false" multiValued="false"/>
```
Comment (eddturner) [inline solr-cores/src/main/cores/annotation/conf/schema.xml]: Just noticed that this change should also be reflected in any `search.properties` and `application.properties` which refer to fields. E.g., `annotation.terms.query.compatible.fields` property (and possibly `search.return.fields`?)

Comment (rantunesboreas) [inline solr-cores/src/main/cores/annotation/conf/schema.xml]: Done

**Pull request merged**